admin.md

```diff
@@ -1,112 +1,145 @@
-# HRI Administration
-
-HRI Administration tasks can be split up into two categories: 
-  1. [Creation of new HRI Tenants](#create-a-new-tenant)
-      1. [Set Up Curl with ElasticSearch](#set-up-curl-with-elasticsearch)
-      2. [Set the Default Index Template](#set-the-default-index-template)
-      3. [Create a new Tenant in ElasticSearch](#create-a-new-tenant-in-elasticsearch)
-  2. [Onboarding Data Integrators](#onboarding-new-data-integrators)
-      1. [Using the Management API Stream Endpoints](#using-the-management-api-stream-endpoints)
-  3. [HRI Management API Keys](#hri-management-api-keys)
-   
-## Create a new Tenant
+# Administration
 
-#### Set Up Curl with ElasticSearch
-  1. Note: this requires the use of the [command-line cURL](https://github.com/curl/curl) tool
-  2. In your IBM Cloud [ElasticSearch (Document Store)](glossary.md#elasticsearch) account, you will need to have either:
-      * an existing Service Credential you can access  OR
-      * create a new Service Credential that you will use below for the `$USERNAME` and  `$PASSWORD` needed to run the ElasticSearch REST commands
-    
-To create a new Service Credential:
+**Overview of administrative tasks for Health Record Ingestion:** 
+
+[Creating new tenants](#creating-new-tenants)
+
+      - [Setting up curl with Elasticsearch](#setting-up-curl-with-elasticsearch)
+      - [Setting the default index template](#setting-the-default-index-template)
+      - [Creating a new tenant in Elasticsearch](#create-a-new-tenant-in-elasticsearch)
```

Comment:
This Syntax of having the dash "-" before the page link here does not Work. The link shows up show up like this...
![badLinks](https://user-images.githubusercontent.com/2658189/99840282-41627080-2b3a-11eb-9d29-4eab58bad09b.png). 

They will need some other formatting choice to make them bullet points.

Reply:
I've removed bulleting.